### PR TITLE
Moving to stop downloading dumb init and supervisord from the internet -requirements

### PR DESCRIPTION
**Pull request by mohammedzee1000**
This PR aims to get supervisord to be able to build without pulling things from the internet
Signed-off-by: Mohammed Zeeshan Ahmed <mohammed.zee1000@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,20 +3,47 @@
...
+
+# Actual image
+
 FROM registry.access.redhat.com/ubi7/ubi
 
 ENV SUPERVISORD_DIR /opt/supervisord
 
-ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 ${SUPERVISORD_DIR}/bin/dumb-init 
+COPY --from=dumbinitbuilder /tmp/dumb-init-src/dumb-init ${SUPERVISORD_DIR}/bin/dumb-init
+#ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 ${SUPERVISORD_DIR}/bin/dumb-init
```
Comment (kadel) [inline Dockerfile]: we can remove this line

File: Dockerfile [hunk 2]
```diff
@@ -3,20 +3,47 @@
 # in order to appropriately inject the supervisord binary into the application container.
 #
 
+# SUPERVISORD
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS supervisordbuilder
```
Comment (kadel) [inline Dockerfile]: I guess that this is supposed to be a "upstream" Dockerfile.
This image can't be downloaded without first logging to ci.opesnhfit.org

File: scripts/update-vendor.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+update_fix_permissions() {
+    echo "Downloading fix permissions script"
+    FIX_PERMISSIONS_SCRIPT="${REQUIREMENTS_DIR}/fix-permissions";
+    curl -o $FIX_PERMISSIONS_SCRIPT https://raw.githubusercontent.com/sclorg/s2i-base-container/master/core/root/usr/bin/fix-permissions;
+    chmod +x $FIX_PERMISSIONS_SCRIPT;
+}
+
+download_tar() {
+    DWN=$1
```
Comment (kadel) [inline scripts/update-vendor.sh]: can we have more descriptive names for this variables¨?

File: Dockerfile (hunk 1 above)
Comment (mohammedzee1000) [reply Dockerfile]: Will do

File: scripts/update-vendor.sh (hunk 3 above)
Comment (mohammedzee1000) [reply scripts/update-vendor.sh]: ok

**Review by cdrage — APPROVED** [review]
/lgtm

**Pull request merged**